### PR TITLE
AWS: Use pagination for DescribeImages call

### PR DESCRIPTION
**Pull request by johngmyers**
New AWS feature!

### CODE REVIEW

File: cloudmock/aws/mockec2/images.go [hunk 1]
```diff
@@ -46,18 +46,18 @@ func (m *MockEC2) DescribeImagesWithContext(aws.Context, *ec2.DescribeImagesInpu
 	panic("Not implemented")
 }
 
-func (m *MockEC2) DescribeImages(request *ec2.DescribeImagesInput) (*ec2.DescribeImagesOutput, error) {
+func (m *MockEC2) DescribeImagesPages(request *ec2.DescribeImagesInput, callback func(output *ec2.DescribeImagesOutput, b bool) bool) error {
```
Comment (justinsb) [inline cloudmock/aws/mockec2/images.go:49]: Can we use DescribeImagesPagesWithContext (rather than having to introduce it later)

**Comment by justinsb** [comment]
Looks good.  One nit to start using the context form right away rather than introducing it later (because we have to mock it as well).  Are we actually fetching lots of images though?  Any way we could avoid doing so (e.g. can we sort by timestamp?)

**Comment by johngmyers** [comment]
I don't see any way to request sorting. I don't think it's common to fetch a large number of images, but it seems possible someone might have a huge number of versions of an image.

**Review by justinsb — APPROVED** [review]

**Comment by justinsb** [comment]
Thanks - lgtm!

/approve
/lgtm

**Pull request merged**